removed incorrect description of guest experience (#3648)
mattermost/docs: source/deployment/guest-accounts.rst

3. Complete the **Guest Filter** field.
4. Choose **Save**.

If a Mattermost Guest user has the guest role removed in the AD/LDAP system, the synchronization process
will not automatically promote them to a member user role. This is done manually via **System Console > User Management**. If a
member user has the **Guest Attribute** added, the synchronization processes will automatically demote the member user to the guest role.

> If a guest logs in without any channels assigned, they are presented with a blank page - aside from the name of the server and an alert advising them to contact an administrator as they have no channels assigned. 

Reviewer: ```suggestion
When a guest logs in without having any channels assigned to their account, they are presented with an alert advising them to contact an administrator. 
```

3. Complete the **Guest Attribute** field.
4. Choose **Save**.

> If a guest logs in without any channels assigned, they are presented with a blank page - aside from the name of the server and an alert advising them to contact an administrator as they have no channels assigned. 

Reviewer: ```suggestion
When a guest logs in without having any channels assigned to their account, they are presented with an alert advising them to contact an administrator. 
```
